doc/cli.md

```diff
@@ -37,11 +37,14 @@ $ datahike query '[:find ?n . :where [?e :name ?n]]' db:myconfig.edn
 "Linus" # prints the name
 ```
 
+Note that the `conn:<file>` argument to `transact` comes before the transaction
+value(s), whereas the `db:<file>` argument to `query` comes after the query value,
```

Comment:
The reason for this is that you can pass more than one db, in fact an arbitrary number of different data sources into the query engine. This is a core benefit of using datahike, so maybe we should also highlight that here.

Reply:
Good point, I emphasized the connection by combining the two paragraphs. Let me know what you think. Also fixed a couple punctuation errors that I noticed.

Reply:
You can pass also files with `edn:/some/path/file.edn` or `json:/some/path/file.json`. My idea in general is to keep the way inputs are represented medium specific, while the API should always be the same. For a command line client it was necessary to provide a small DSL for file loading and dereferencing. 

This is also mentioned here:
https://github.com/replikativ/datahike/blob/main/src/datahike/cli.clj#L20

Reply:
As a sidenote: Files are fairly reasonable as an abstraction on Unix, but maybe there are better ways to design this interface.

Reply:
Hmmm interesting. Well now that you got me thinking about it, one thing this DSL rules out is using Bash [process substitution](https://www.gnu.org/software/bash/manual/html_node/Process-Substitution.html#Process-Substitution) to specify file contents ad-hoc, like:

`datahike transact <(echo '{:store {:backend :mem :id "mydb"}}') ...`

I don't know when I would use that over a regular file, and if you wanted to support multiple formats with it you would need to pair it with an option or something (`datahike transact --edn <(...)`). Just thinking out loud, since you mention Unix and files.

Reply:
Transact does support to read in transaction data from STDIN already, but that is only for transact. I need to think about it more. I think you could pipe through files, right?